DEVELOPER.md

```diff
@@ -60,7 +60,7 @@ git config blame.ignoreRevsFile .git-blame-ignore-revs
 
 ### Python
 
-FloPy supports several recent versions of Python, loosely following [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html#implementation).
+This project has historically aimed to support several recent versions of Python, loosely following [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html#implementation). In current and future development this window may narrow to follow [SPEC 0](https://scientific-python.org/specs/spec-0000/#support-window) instead.
```

Comment:
NEP 29 is pretty much the same as SPEC 0 (both have about 3 concurrent releases of Python). I'm guessing that NumPy dropped it in favour of SPEC 0 due to a wider-supported specification. I recall the the historic FloPy support was following the official Python supported versions, where the old ones are "security" fixes. Maybe this rephrase?
```suggestion
This project has historically aimed to support a wide range of [Python versions](https://devguide.python.org/versions/). In current and future development this window may narrow to follow [SPEC 0](https://scientific-python.org/specs/spec-0000/#support-window) instead.
```
further suggestions are obviously welcome.

Reply:
thanks @mwtoews, applied.